pkgs/development/python-modules/urwidtrees/default.nix

```diff
@@ -3,20 +3,21 @@
 , fetchFromGitHub
 , glibcLocales
 , urwid
+, mock
 }:
 
 buildPythonPackage rec {
   pname = "urwidtrees";
-  version  = "1.0.2";
+  version  = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "pazz";
     repo = "urwidtrees";
     rev = version;
-    sha256 = "1n1kpidvkdnsqyb82vlvk78gmly96kh8351lqxn2pzgwwns6fml2";
+    sha256 = "sha256-yGSjwagCd5TiwEFtF6ZhDuVqj4PTa5pVXhs8ebr2O/g=";
   };
 
-  propagatedBuildInputs = [ urwid ];
+  propagatedBuildInputs = [ urwid mock ];
 
   checkInputs = [ glibcLocales ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ urwid ];

  checkInputs = [ glibcLocales mock ];
```

Reply:
actually it is required https://github.com/pazz/urwidtrees/blob/1.0.3/setup.py#L17 . It is optional on master though.

Reply:
it will still be present during the build, and it does in fact build

```
[nix-shell:~/.cache/nixpkgs-review/pr-103149-1]$ nix-build ./nixpkgs/ -A python3Packages.urwidtrees
/nix/store/dmgiwb6jkd7y7hr7k034xa0l7ciccr63-python3.8-urwidtrees-1.0.3

[nix-shell:~/.cache/nixpkgs-review/pr-103149-1]$ cd nixpkgs/
COPYING  default.nix  doc  flake.nix  lib  maintainers  nixos  pkgs  README.md

[nix-shell:~/.cache/nixpkgs-review/pr-103149-1/nixpkgs]$ git diff
diff --git a/pkgs/development/python-modules/urwidtrees/default.nix b/pkgs/development/python-modules/urwidtrees/default.nix
index 442b4ce3ad8..737f63a6801 100644
--- a/pkgs/development/python-modules/urwidtrees/default.nix
+++ b/pkgs/development/python-modules/urwidtrees/default.nix
@@ -17,9 +17,9 @@ buildPythonPackage rec {
     sha256 = "sha256-yGSjwagCd5TiwEFtF6ZhDuVqj4PTa5pVXhs8ebr2O/g=";
   };

-  propagatedBuildInputs = [ urwid mock ];
+  propagatedBuildInputs = [ urwid ];

-  checkInputs = [ glibcLocales ];
+  checkInputs = [ glibcLocales mock ];
   LC_ALL="en_US.UTF-8";

   meta = with stdenv.lib; {
```